README.md

```diff
@@ -15,6 +15,7 @@ Find out more about TCES by visiting the following page: https://toronto-jobs.or
 ## Contributors
 - Daniel Dervishi
 - Jordan Janakievski
+- Tanmay Shah
```

Comment:
Please refer to the linear workflow wiki document and assure that you have completed all the items listed in the linear ticket. 

Linear Workflow Document: https://github.com/uoftblueprint/tces/wiki/Git-Workflow-with-Linear-Integration